modules/installation/proc-install-rhdh-osd-gcp-helm.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+// assembly-install-rhdh-osd-gcp.adoc
+
+[id="proc-install-rhdh-osd-gcp-helm_{context}"]
+= Installing {product} on {osd-short} on {gcp-short} using the Helm Chart
+
+You can install {product-short} on {osd-short} on {gcp-short} using the {product} Helm Chart.
+
+.Prerequisites
+* You have a valid {gcp-short} account.
+* Your {osd-short} cluster is running on {gcp-short}. For more information, see link:https://docs.redhat.com/en/documentation/openshift_dedicated/4/html/installing_accessing_and_deleting_openshift_dedicated_clusters/osd-creating-a-cluster-on-gcp[Creating a cluster on GCP] in {osd-brand-name} documentation.
+* You have administrator access to {osd-short} cluster and {gcp-short} project.
```

Comment:
Administrator access is not needed to install using Helm.

Reply:
Removed the prereq.